### PR TITLE
camera calibration without images

### PR DESCRIPTION
**Pull request by RohitShaw340**

### CODE REVIEW

File: StereoCalibration/calibration.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+cv_file.open("stereoMap.xml", cv2.FileStorage_READ)
+
+stereoMapL_x = cv_file.getNode("stereoMapL_x").mat()
+stereoMapL_y = cv_file.getNode("stereoMapL_y").mat()
+stereoMapR_x = cv_file.getNode("stereoMapR_x").mat()
+stereoMapR_y = cv_file.getNode("stereoMapR_y").mat()
+
+
+def undistortRectify(frameR, frameL):
+
```
Comment (0Mayank) [inline StereoCalibration/calibration.py:18]: Fix blank line

**Review by 0Mayank — DISMISSED** [review]

File: StereoCalibration/calibration.py (hunk 1 above)
Comment (RohitShaw340) [reply StereoCalibration/calibration.py:18]: ye rahega

Comment (RohitShaw340) [reply StereoCalibration/calibration.py:18]: galat blame mat kar

Comment (0Mayank) [reply StereoCalibration/calibration.py:18]: Shi blame h

**Pull request merged**